adal/build.gradle

```diff
@@ -125,12 +125,7 @@ dependencies {
     }
 
     // 'dist' flavor dependencies
-    // Once common is deployed to maven, uncomment the below line...
-    // distImplementation "com.microsoft.identity:common:0.0.1"
-    // and delete the 3 following lines....
-    distImplementation(project(":common")) {
-        transitive = false
-    }
+     distImplementation "com.microsoft.identity:common:0.0.2"
```

Comment:
Oops. This declaration should also ignore transitives

```groovy
distImplementation("com.microsoft.identity:common:0.0.2") {
    transitive = false
}
```

Reply:
My bad :unamused: Updated it. Thanks @iambmelt